lib/screens/play_game_screen.dart

```diff
@@ -0,0 +1,230 @@
+        await _writeLedCharacteristic([0, 255, 0, 255]);
+      }
+    });
+  }
+
+  Future<void> _writeLedCharacteristic(List<int> value) async {
+    await widget.ledCharacteristic?.write(value);
+  }
+
+  void _navigateBack() {
+    if (_tapCount == 0) {
+      Navigator.push(
+        context,
+        MaterialPageRoute(
+          builder: (context) => ChooseGameScreen(
+            sensorCharacteristic: widget.sensorCharacteristic,
+            ledCharacteristic: widget.ledCharacteristic,
+          ),
+        ),
+      );
+    }
+  }
+
+  void _decreaseTapCount() {
+    if (_timerValue == 0) {
+      setState(() {
+        _tapCount--;
+      });
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    final loc = AppLocalizations.of(context)!;
+
+    setState(() {
+      if (_timerValue > 0) {
+        _gameTitle = loc.startGameText;
+      } else {
+        _gameTitle = loc.tapsLeft;
+      }
+    });
+
+    return Container(
+      decoration: const BoxDecoration(
+        color: Colors.transparent,
+        image: DecorationImage(
+          image: AssetImage(Assets.chooseGameBackgroundImageAsset),
+          fit: BoxFit.fill,
+        ),
+      ),
+      child: Column(
+        children: [
+          Align(
+            alignment: Alignment.topCenter,
+            child: Padding(
+              padding: const EdgeInsets.only(top: 50.0),
+              child: Container(
+                decoration: const BoxDecoration(
+                  color: Color.fromRGBO(255, 255, 255, 0),
+                ),
+                child: SvgPicture.asset(
+                  Assets.logoAsset,
+                  width: 138,
+                ),
+              ),
+            ),
+          ),
+          Expanded(
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              crossAxisAlignment: CrossAxisAlignment.center,
+              children: [
+                Padding(
+                  padding: const EdgeInsets.only(top: 20.0),
+                  child: Text(
+                    _gameTitle,
+                    style: theme.textTheme.headlineMedium,
+                    textAlign: TextAlign.center,
+                  ),
+                ),
+                Expanded(
+                  child: GestureDetector(
+                    onTap: _decreaseTapCount,
+                    child: Align(
+                      alignment: Alignment.center,
+                      child: GestureDetector(
+                        onTap: _navigateBack,
+                        child: Text(
+                          _timerValue > 0
+                              ? _timerValue.toString()
+                              : _tapCount == 0
+                                  ? "Hihhihhii. Voitit pelin!!1"
+                                  : _tapCount.toString(),
+                          style: theme.textTheme.headlineMedium?.copyWith(
+                            fontSize: (2 *
```

Comment:
Use non-null assertions, this kind of code does not make sense. As long as `headlineMedium` is defined when constructing `MaterialApp`, which it is, its not gonna be null.